packs/autoscale/actions/workflows/asg_delete.yaml

```diff
@@ -15,116 +15,134 @@ workflows:
+          key: 'asg.<% $.name %>.loadbalancer_id'
         on-success:
           - get_dns_zone_id
       get_dns_zone_id:
         action: st2.kv.get
         input:
-          key: 'asg.{$.name}.dns_zone_id'
+          key: 'asg.<% $.name %>.dns_zone_id'
         publish:
-          dns_zone_id: '{$.get_dns_zone_id.result}'
+          dns_zone_id: '<% $.get_dns_zone_id.result %>'
         on-success:
           - delete_dns_zone
       delete_dns_zone:
         action: rackspace.delete_dns_zone
         input:
-          zone_id: "{$.dns_zone_id}"
+          zone_id: "<% $.dns_zone_id %>"
         on-success:
           - delete_dns_zone_key
       delete_dns_zone_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.dns_zone_id'
+          key: 'asg.<% $.name %>.dns_zone_id'
         on-success:
           - chatops_clear_memory
       delete_min_nodes_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.min_nodes'
+          key: 'asg.<% $.name %>.min_nodes'
         on-success:
           - delete_max_nodes_key
       delete_max_nodes_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.max_nodes'
+          key: 'asg.<% $.name %>.max_nodes'
         on-success:
           - chatops_clear_memory
       delete_vm_size_id_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.vm_size_id'
+          key: 'asg.<% $.name %>.vm_size_id'
         on-success:
           - delete_vm_image_id_key
       delete_vm_image_id_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.vm_image_id'
+          key: 'asg.<% $.name %>.vm_image_id'
         on-success:
           - delete_domain_key
       delete_domain_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.domain'
+          key: 'asg.<% $.name %>.domain'
         on-success:
           - delete_server_role
       delete_expand_by_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.expand_by'
+          key: 'asg.<% $.name %>.expand_by'
         on-success:
           - delete_expand_delay_key
       delete_expand_delay_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.expand_delay'
+          key: 'asg.<% $.name %>.expand_delay'
         on-success:
           - delete_last_expand_timestamp
       delete_last_expand_timestamp:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.last_expand_timestamp'
+          key: 'asg.<% $.name %>.last_expand_timestamp'
         on-success:
           - delete_last_deflate_timestamp
       delete_deflate_by_key:
         action: st2.kv.delete
         input:
-          key: 'asg.{$.name}.deflate_by'
+          key: 'asg.<% $.name %>.deflate_by'
         on-success:
           - delete_deflate_delay_key
       delete_deflate_delay_key:
         action: st2.kv.delete
```

Comment:
Ah! Good call. I was wondering how to make all the key writes a bit cleaner. Also have some thoughts on how to make multiple calls as well

-James

> On Feb 26, 2015, at 11:22 PM, Dmitri Zimin(e) <notifications@github.com> wrote:
> 
> In packs/autoscale/actions/workflows/asg_delete.yaml:
> 
> >          on-success:
> >            - delete_last_deflate_timestamp
> >        delete_deflate_by_key:
> >          action: st2.kv.delete
> >          input:
> > -          key: 'asg.{$.name}.deflate_by'
> > +          key: 'asg.<% $.name %>.deflate_by'
> >          on-success:
> >            - delete_deflate_delay_key
> >        delete_deflate_delay_key:
> >          action: st2.kv.delete
> consider one-liner: st2.kv.delete key="'asg.<% $.name %>.deflate_delay'"
> 
> —
> Reply to this email directly or view it on GitHub.
>